app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -937,6 +937,26 @@ class Settings(private val appContext: Context) : PreferencesHolder {
         0
     )
 
+    var installedAddonsCount by intPreference(
+        appContext.getPreferenceKey(R.string.pref_key_installed_addons_count),
+        0
+    )
+
+    var installedAddonsList by stringPreference(
+        appContext.getPreferenceKey(R.string.pref_key_installed_addons_list),
+        default = ""
+    )
+
+    var enabledAddonsCount by intPreference(
+        appContext.getPreferenceKey(R.string.pref_key_enabled_addons_count),
+        0
+    )
+
+    var enabledAddonsList by stringPreference(
+        appContext.getPreferenceKey(R.string.pref_key_enabled_addons_list),
+        default = ""
+    )
```

Comment:
Let's start adding some kDoc describing the variables we have in `Settings`. Something to describe what they hold in the preference and what they are used for.

Reply:
Sure, I'll add them.  Thanks,